code/modules/crafting/smeltery.dm

```diff
@@ -4,43 +4,59 @@
 	name = "Smith Broadsword"
 	result = /obj/item/weapon/smithed_sword
 	reqs = list(/obj/item/weapon/grown/log = 1,
-				/obj/item/weapon/mold_result/blade = 1)
+				/obj/item/stack/sheet/leather = 1,
+				/obj/item/weapon/mold_result/blade = 1,
+				/obj/machinery/anvil)
 	parts = list(/obj/item/weapon/mold_result/blade = 1)
 	time = 40
 	category = CAT_SMITH
 
+/datum/crafting_recipe/warhammer
+	name = "Smith Warhammer"
+	result = /obj/item/weapon/twohanded/smithed_warhammer
+	reqs = list(/obj/item/weapon/grown/log = 1,
+				/obj/item/weapon/mold_result/hammer_head = 1,
+				/obj/machinery/anvil)
+	parts = list(/obj/item/weapon/mold_result/hammer_head = 1)
+	time = 40
+	category = CAT_SMITH
+
 /datum/crafting_recipe/pickaxe
-	name = "Smithed Pickaxe"
+	name = "Smith Pickaxe"
 	result = /obj/item/weapon/pickaxe/smithed_pickaxe
 	reqs = list(/obj/item/weapon/grown/log = 1,
-				/obj/item/weapon/mold_result/pickaxe_head = 1)
+				/obj/item/weapon/mold_result/pickaxe_head = 1,
+				/obj/machinery/anvil)
 	parts = list(/obj/item/weapon/mold_result/pickaxe_head = 1)
 	time = 40
 	category = CAT_SMITH
 
 /datum/crafting_recipe/shovel
-	name = "Smithed Shovel"
+	name = "Smith Shovel"
 	result = /obj/item/weapon/shovel/smithed_shovel
 	reqs = list(/obj/item/weapon/grown/log = 1,
-				/obj/item/weapon/mold_result/shovel_head = 1)
+				/obj/item/weapon/mold_result/shovel_head = 1,
+				/obj/machinery/anvil)
 	parts = list(/obj/item/weapon/mold_result/shovel_head = 1)
 	time = 40
 	category = CAT_SMITH
 
 /datum/crafting_recipe/smith_armor
 	name = "Smith Armor"
 	result = /obj/item/clothing/suit/armor/vest/dwarf
-	reqs = list(/obj/item/weapon/grown/log = 1,
-				/obj/item/weapon/mold_result/armor_plating = 1)
+	reqs = list(/obj/item/stack/sheet/leather = 4,
+				/obj/item/weapon/mold_result/armor_plating = 1,
+				/obj/machinery/anvil)
 	parts = list(/obj/item/weapon/mold_result/armor_plating = 1)
 	time = 40
 	category = CAT_SMITH
 
 /datum/crafting_recipe/smith_helmet
 	name = "Smith Helmet"
 	result = /obj/item/clothing/head/helmet/dwarf
-	reqs = list(/obj/item/weapon/grown/log = 1,
-				/obj/item/weapon/mold_result/helmet_plating = 1)
+	reqs = list(/obj/item/stack/sheet/leather = 2,
```

Comment:
I was wrong in assuming cat's butcher_result gave their respective hide, but that's a super easy fix so technically they will as soon as I add that. They have the puddle for wetting and can make a drying rack so leather should be more than feasible as long as they don't kill off all their livestock.